Automattic/jetpack
# Update sharing button styles

## Proposed changes:

This PR is one of several steps that we'll take to update the look and feel of the sharing icons. You can see a full list of updates that need to be made [here](https://github.com/Automattic/jetpack/issues/27817). There are two primary sets of changes here:

## Front-end

On the front-end of your site, this is the before and after:

### Before

![CleanShot 2023-02-09 at 12 09 52](https://user-images.githubusercontent.com/5634774/217887493-9d3b572f-5dda-4e9e-8919-e71265caae81.png)

### After

![CleanShot 2023-02-09 at 12 08 53](https://user-images.githubusercontent.com/5634774/217887464-02821b63-6f28-49c1-a4ff-dbb16e0856dd.png)

## Admin settings

When you go to `/wp-admin/options-general.php?page=sharing` here's the before & after:

### Before

<img width="1052" alt="CleanShot 2023-02-13 at 14 42 06@2x" src="https://user-images.githubusercontent.com/5634774/218558344-25e1b371-239f-4277-a08c-a11ecdcf885c.png">

### After

<img width="1059" alt="CleanShot 2023-02-13 at 14 41 18@2x" src="https://user-images.githubusercontent.com/5634774/218558169-b9f5a685-15a1-4fce-9b65-c5a80d57e316.png">

## Jetpack product discussion
- #27817
- pe7F0s-i3-p2
- pe7F0s-qg-p2

## Does this pull request change what data or activity we track or use?
Nope.

## Testing instructions:
- Follow instructions to [set up JP locally](https://github.com/Automattic/jetpack/blob/trunk/docs/quick-start.md) (with jurassic.tube proxy), or use jurassic ninja
- Ensure that you have sharing buttons activated
- Ensure that you have at least one published blog post
- Visit the blog post

## Related
- #27817
- #28876
- https://github.com/Automattic/wp-calypso/pull/73126
